### PR TITLE
OCPBUGS-7150: fix the mount for cluster ca certs

### PR DESCRIPTION
**Pull request by PratikMahajan**

### CODE REVIEW

File: controllers/new.go [hunk 1]
```diff
@@ -637,7 +637,7 @@ func (k *kubeResources) newGraphBuilderVolumeMounts(instance *cv1.UpdateService)
 		vm = append(vm, corev1.VolumeMount{
 			Name:      NameClusterTrustedCAVolume,
 			ReadOnly:  true,
-			MountPath: "/etc/pki/ca-trust/extracted/pem/cluster-ca-bundle.crt",
+			MountPath: "/etc/pki/ca-trust/extracted/pem/ca-bundle.crt",
```
Comment (LalatenduMohanty) [inline controllers/new.go:640]: We still use the hard coded path here. Can we use a variable instead as we have the file name in `NameClusterCertConfigMapKey`?

**Review by LalatenduMohanty — CHANGES_REQUESTED** [review]

**Comment by PratikMahajan** [comment]
/override operator-e2e 
Using older version of cincinnati which does not have a multi-arch fix

**Review by PratikMahajan — COMMENTED** [review]
/override ci/prow/operator-e2e

**Pull request merged**